Looked things over and all looks good. Are we still wanting to do one massive config file? In which case it may make sense to move the `mapping/config.yaml` to `test` or some other higher level folder.

I deleted references-test and tried running tests. Had this error pop up on yeast genome. Have lab meeting now, but will try trouble shooting after that.

```
Error in job download_fasta while creating output file /data/LCDB/references-test/sacCer3/sacCer3_default.fa.gz.
RuleException:
TypeError in line 66 of /gpfs/gsfs4/users/LCDB/users/fearjm/lcdb-workflows/workflows/references/references.snakefile:
filename must be a str or bytes object, or a file
  File "/gpfs/gsfs4/users/LCDB/users/fearjm/lcdb-workflows/workflows/references/references.snakefile", line 110, in __rule_download_fasta
  File "/gpfs/gsfs4/users/LCDB/users/fearjm/lcdb-workflows/workflows/references/references.snakefile", line 66, in download_and_postprocess
  File "/gpfs/gsfs4/users/LCDB/users/fearjm/lcdb-workflows/workflows/references/sacCer3.py", line 11, in fasta_postprocess
  File "/home/fearjm/opt/miniconda3/envs/lcdb-workflows-fearjm-env/lib/python3.5/tarfile.py", line 1562, in open
  File "/home/fearjm/opt/miniconda3/envs/lcdb-workflows-fearjm-env/lib/python3.5/tarfile.py", line 1660, in bz2open
  File "/home/fearjm/opt/miniconda3/envs/lcdb-workflows-fearjm-env/lib/python3.5/bz2.py", line 102, in __init__

Traceback (most recent call last):
  File "test/run_test.py", line 155, in <module>
    sp.check_call(['bash', script_name])
  File "/home/fearjm/opt/miniconda3/lib/python3.5/subprocess.py", line 584, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['bash', 'lcdb-workflows-submit-8307974e.sh']' returned non-zero exit status 1
```

lcdb-init.py ran cleanly (but needed chmod a+x).

--- a/workflows/references/sacCer3.py
+++ b/workflows/references/sacCer3.py
@@ -8,6 +8,10 @@ def fasta_postprocess(origfn, newfn):
     The fasta from UCSC comes as a tarball of fastas. So we extract them all to
     a temp directory and then cat them all together into the final fa.gz file.
     """
+    assert (
+        (isinstance(origfn, list)) and (len(origfn) == 1)
+    ), 'unexpected input: %s' % origfn
+    origfn = origfn[0]
     t = tarfile.open(origfn)
     shell('mkdir -p {origfn}.tmp')
     t.extractall(origfn + '.tmp')

